src/open_inwoner/templates/pages/profile/contacts/list.html

```diff
@@ -8,6 +8,47 @@ <h1 class="h1" id="title">
     {% button href="accounts:contact_create" text=_("Nieuw contact toevoegen") icon="person" primary=True %}
 </h1>
 
+<!--List of contact requests: receiver's view-->
+{% if pending_approval or pending_approval_list %}
+    <div class="approval{% if pending_approval %} approval--solo{% endif %}">
+        <div class="approval__header">
+            <h2 class="h2">{% trans "U bent toegevoegd als contactpersoon" %}</h2>
+            <p class="p">{% trans "By accepting you agree on sharing your personal data (first name, last name and phonenumber)." %}</p>
```

Comment:
I don't see this sentence it in the design (https://projects.invisionapp.com/share/Y211V6JRHPC4#/screens/470424529). It the design updated somewhere else?